types/primitive.yaml

```
@@ -26,7 +26,7 @@ schemas:
 
   Uint64:
     type: string
-    example: "1213123123123123123123123122"
+    example: "18446744073709551615"
```

```suggestion
    example: "1"
```
I don't think we should put max value as example as it is suggested in query and path params. I also should not put random value, tnx for noticing